package.json

```diff
@@ -25,7 +25,7 @@
     "axios": "^0.19.0",
     "cli-ux": "^5.2.1",
     "command-exists": "^1.2.8",
-    "eclipse-che": "git://github.com/eclipse/che#9976bb0",
+    "eclipse-che": "git://github.com/eclipse/che#master",
```

Comment:
I think that it makes sense to use `che#master` here on `chectl#master` branch since we plan to release chectl as we do with che (at every sprint) and we should then use `che#$release-num` on `chectl#release-num` branch.

Reply:
cc @benoitf

Reply:
I'm also fine for master. I was using it but @l0rd said to use a fixed sha-1

To provide more content here: what I believe is that chectl should have helm charts for the nightlies and also embed the helm charts for every previous Che releases. (like the previous docker-based CLI)
So it can work smoothly on every Che server (it will provide the associate charts for the matching version)
And a chectl needs to be released every night as well.